lua/lang/italian/shinigami.lua

```diff
@@ -0,0 +1,10 @@
+L = LANG.GetLanguageTableReference("italian")
+
+L[SHINIGAMI.name] = "Shinigami"
+L["info_popup_" .. SHINIGAMI.name] = [[Sei uno Shinigami! Prova ad uccidere i terroristi malvagi!]]
+L["body_found_" .. SHINIGAMI.abbr] = "Era uno Shinigami."
+L["search_role_" .. SHINIGAMI.abbr] = "Questa persona era uno Shinigami!"
+L["target_" .. SHINIGAMI.name] = "Shinigami"
+L["ttt2_desc_" .. SHINIGAMI.name] = [[Lo Shinigami è un Innocente (che collabora con gli altri innocenti) ed il suo obiettivo è quello di uccidere tutti i ruoli malvagi ^^ Lo Shinigami può vedere il nome dei suoi nemici.]]
+
+L["ttt2_shinigami_chat_jammed"] = "The Chat is jammed! You can't use the chat as a respawned Shinigami."
```

Comment:
Would be awesome, if you could translate this aswell

Reply:
Woops, sorry, just noticed I forgot to translate that. Updated it!